templates/locale/de/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,283 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
```

Comment:
Let's remove DE files for now and add translations for only FR

Reply:
Is this resolved now?

Reply:
@andrewtpham yes I have removed DE locale file :+1: